elements/pfe-sass/src/variables/_3-colors.scss

```diff
@@ -70,59 +70,59 @@ $pfe-color--ui-disabled--text--hover:           $pfelements--gray !default;
 // Lightest group
 $pfe-color--surface--lightest:                  $pfe-color--white !default;
 $pfe-color--surface--lightest--text:            $pfe-color--text !default;
-$pfe-color--surface--lightest--link:            $pfe-color--ui-link !default;
-$pfe-color--surface--lightest--link--visited:   $pfe-color--ui-link--visited !default;
-$pfe-color--surface--lightest--link--hover:     $pfe-color--ui-link--hover !default;
-$pfe-color--surface--lightest--link--focus:     $pfe-color--ui-link--focus !default;
+$pfe-color--surface--lightest--ui-link:            $pfe-color--ui-link !default;
```

Comment:
I updated this because I was running into issues building the logic in the mixins to apply the broadcast colors.

Reply:
This is the naming for the broadcasted variable: `--pfe-broadcasted--color--ui-link--visited`, it was easier to be consistent in our link references and since we're already using broadcasted in a lot of places, I thought adding it would be easier to the sass variables instead.

Reply:
ah, the problemo is actually that the generated CSS is now looking for variables that aren't in the theme (+/- `-ui`). From pfe-accordion-header.css:
```
:host button:hover {
  --pfe-accordion--accent: var(--pfe-theme--color--surface--lightest--ui-link, #06c);
}
```

From [advanced-theme.css](https://gitlab.corp.redhat.com/uxdd/redhat-theme/blob/2.2.1/dist/advanced-theme.css)
```
--pfe-theme--color--surface--lightest--link: #0066cc;
```